mmdet/models/backbones/resnet.py

```diff
@@ -302,23 +356,8 @@ def forward(self, x):
 
     def train(self, mode=True):
```

Comment:
If we set eval mode in `build_norm_layer`, then this method is not necessary any more.